vslm/object_manager.go

```diff
@@ -258,6 +258,37 @@ func (m ObjectManager) RegisterDisk(ctx context.Context, path, name string) (*ty
 	return &res.Returnval, nil
 }
 
+func (m ObjectManager) ExtendDisk(ctx context.Context, id string, newCapacityInMB int64, ds mo.Reference) (*object.Task, error) {
```

Comment:
The other methods here with similar params all have `ds` as the 2nd param and `id` as the third param. Can we do the same for these new methods?